### PR TITLE
Update Release.toml to v1.10.0

### PR DESCRIPTION
**Pull request by etungsten**
**Issue number:**
N/A

**Description of changes:**
We originally did this in e06785eb, but we had to make a point release for v1.9.0.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by stmcginnis — APPROVED** [review]

File: Release.toml [hunk 1]
```diff
@@ -142,3 +142,5 @@ version = "1.9.1"
 ]
 "(1.9.0, 1.9.1)" = [
 ]
+"(1.9.1, 1.10.0)" = [
```
Comment (zmrow) [inline Release.toml]: Nit: Could we put the square brackets on a single line to stay consistent with previous empty migration lists? :)

**Comment by etungsten** [comment]
Push above addresses @zmrow 's comment

**Review by bcressey — APPROVED** [review]

**Review by zmrow — APPROVED** [review]
:champagne:

**Comment by etungsten** [comment]
Push above rebases onto develop and resolves conflicts

**Review by stmcginnis — APPROVED** [review]

**Pull request merged**